{
  "modified": "2025-08-28T07:11:30Z",
  "published": "2025-08-28T07:11:30Z",
  "schema_version": "1.5.0",
  "id": "",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "peloton-client123",
        "purl": "pkg:pypi/peloton-client123"
      },
      "versions": [
        "0.8.10"
      ]
    }
  ],
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "e04f5bb9780de3541f853306bdcf9525bc99ed129b897a2a1885948d378df8c4",
        "import_time": "2025-08-29T06:41:48.810999211Z",
        "id": "RLMA-2025-04210",
        "modified_time": "2025-08-28T07:11:30Z",
        "versions": [
          "0.8.10"
        ]
      }
    ]
  }
}
